content/pl/docs/reference/tools.md

```diff
@@ -0,0 +1,54 @@
+---
+reviewers:
+- janetkuo
```

Comment:
Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.